code/game/gamemodes/clock_cult/clock_mobs/_eminence.dm

```diff
@@ -0,0 +1,267 @@
+//The Eminence is a unique mob that functions like the leader of the cult. It's incorporeal but can interact with the world in several ways.
+/mob/camera/eminence
+	name = "\the Emininence"
+	real_name = "\the Eminence"
+	desc = "The leader-elect of the servants of Ratvar."
+	icon = 'icons/effects/clockwork_effects.dmi'
+	icon_state = "eminence"
+	mouse_opacity = MOUSE_OPACITY_OPAQUE
+	move_on_shuttle = TRUE
+	see_in_dark = 8
+	invisibility = INVISIBILITY_OBSERVER
+	layer = FLY_LAYER
+	faction = list("ratvar")
+	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
+	var/static/superheated_walls = 0
+
+/mob/camera/eminence/Initialize()
+	if(SSticker.mode.eminence)
+		return INITIALIZE_HINT_QDEL
+	. = ..()
+
+/mob/camera/eminence/Destroy(force)
+	if(!force && SSticker.mode.eminence)
```

Comment:
Why is this here?

Reply:
I forgot a `== src` at the end there, whoops!